src/aelfrice/hook.py

```
@@ -729,9 +736,28 @@ def _write_hook_audit_record(
+
+    Resolves the same pure env -> kwarg -> TOML resolver that
+    `_split_belief_lines` used to build the block, so within one hook
+    process the recorded value is the one that actually rendered. Returns
+    None (field omitted) if the resolver is unreachable — the audit row is
+    fail-soft and must never take the hook down for a diagnostic field.
+    """
+    try:
+        from aelfrice.retrieval import resolve_order_policy  # noqa: PLC0415
+
+        return resolve_order_policy()
+    except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)